### DISCUSSION TITLE
Using my own demographic data

### DISCUSSION THREAD

**Discussion by Shixing-Ju** [Open-Systems-Pharmacology/Forum#1787] [Questions & Problems]
Hi everyone！
I want to incorporate my own formula for calculating demographic information in PK Sim. Does anyone know how to do it? If you could tell me, I would greatly appreciate it!

**Comment by Yuri05** [comment]
You could e.g. create a population with the "standard" PK-Sim algorithm, export it to CSV and import in MS-Excel.
Then overwrite some parameters using your formulas, export the modified population to CSV and import in PK-Sim as a new population.

**Reply by Shixing-Ju** [reply]
Thank you very much for your reply, it's very useful!
